forms-flow-api/src/formsflow_api/app.py

```diff
@@ -53,13 +53,14 @@ def create_app(
     app.logger = flask_logger
     app.logger = logging.getLogger("app")
 
-    register_log_handlers(
-        app,
-        log_file="logs/forms-flow-webapi.log",
-        when=os.getenv("API_LOG_ROTATION_WHEN", "d"),
-        interval=int(os.getenv("API_LOG_ROTATION_INTERVAL", "1")),
-        backupCount=int(os.getenv("API_LOG_BACKUP_COUNT", "7")),
-    )
+    if app.config["CONFIGURE_LOGS"]:
+        register_log_handlers(
```

Comment:
whether StreamHandler is expected to work if app.config["CONFIGURE_LOGS"] is false

Reply:
@auslin-aot that should still work with default log config without using file right ?

Reply:
![image](https://github.com/AOT-Technologies/forms-flow-ai/assets/95394061/d0a7a8b2-6b18-47bd-b4f5-a7ddf83710c0)
 here the condition if false then `app.logger.handlers = [logs, file_handler]` logs  will work or not

Reply:
Done